### ISSUE TITLE
[SampleCard-FlightDetails.json; Config-Cortana-skills-classic]: The luminosity ratio for the text "Flight to JFK" is 3.67:1 and should be atleast 4.5:1

### ISSUE THREAD

**Issue by carlos-zamora** [microsoft/AdaptiveCards#7373]
https://microsoft.visualstudio.com/OS/_workitems/edit/37985603

**Comment by Raisul123** [comment]
As per new guideline we are enhancing the severity. so we are closing this issue and logged new bug: [[SampleCard-FlightDetails.json; Config-Cortana-skills-classic]: The luminosity ratio for the text "Flight to JFK" is 3.67:1 and should be atleast 4.5:1 · Issue #8478 · microsoft/AdaptiveCards (github.com)](https://github.com/microsoft/AdaptiveCards/issues/8478)
